File: Work/Cinda/cinda-notes/assets/日記_2020_0715/file/b2e/model/DeviceReturnList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Mxic.ITC.PAM.Model.Entity
{
    public class DeviceReturnList
    {
        public decimal Id { get; set; }

        public DateTime? ReturnDate { get; set; }

        public string DeviceReturn { get; set; }

        public string DeviceReturnDesc { get; set; }

        public string XfortReturn { get; set; }

        public string XfortDesc { get; set; }

        public string NbBringout { get; set; }

        public string NbBringoutDesc { get; set; }

        public decimal? SignFormId { get; set; }

        public decimal? AccountId { get; set; }

        public string XfortAssetNo { get; set; }

        public string NbAssetNo { get; set; }

        // FK JOIN

        public string ResignName { get; set; }

        public string ResignEmpNo { get; set; }

        public string ResignDepartment { get; set; }

        public string XfortControl { get; set; }

        public string NbCustody { get; set; }

        public string SignFormNo { get; set; }
    }
}

File: Work/Cinda/cinda-notes/assets/日記_2020_0720_啟單成功/F2E/models/DeviceReturnList.ts
export default class DeviceReturnList {

    // 前端原欄位名稱

    // Department: string;

    // EmpName: string;

    // EmpNo: string;

    // Xfort: string;

    // NB: string

    // Xfort_AssetId: string;

    // NB_AssetId: string;

    // 後端欄位名稱

    Id: string; // Id: string 帳號關閉及設備繳回單 ID

    ReturnDate: Date | string | null; // ReturnDate: Date; // 設備繳回日期 RETURN_DATE

    DeviceReturn: boolean; // DeviceReturn: boolean; // 設備不繳回 DEVICE_RETURN

    DeviceReturnDesc: string;

    XfortReturn: boolean; // Xfort: boolean; // XFORT_RETURN

    XfortDesc: string;

    NbBringout: boolean;  // NB: boolean; // NB_BRINGOUT

    NbBringoutDesc: string;

    SignFormId: number | null;

    AccountId: number | null;

    // Device_AssetId: string // DEVICE_RETURN_DESC

    XfortAssetNo: number;  // NB_AssetId: string;

    NbAssetNo: number; // Xfort_AssetId: string;

    // FK JOIN

    ResignName: string;   // EmpName: string; // 姓名

    ResignEmpNo: string;   // EmpNo: string; // 工號

    ResignDepartment: string; // Department: string; // 部門

    //

    XfortControl: string; // Xfort 是否有Xfort管控?

    NbCustody: string; //  NB 是否有NB攜出保管證?

    SignFormNo: string; // 簽核表單單號
}
